react-app/src/features/forms/post-submission/amend/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { Navigate, useParams } from "react-router-dom";
+import { AmendmentForm as CapitatedForm } from "../../waiver/capitated";
+import { AmendmentForm as ContractingForm } from "../../waiver/contracting";
+import { useGetItem } from "@/api";
+
+export const Amendment = () => {
+  const { id } = useParams();
+  const { data } = useGetItem(id);
```

Comment:
`data` is generic. Can we rename this to `submission` or something more descriptive?